index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <div>
+      <nav class="navbar">
+        <img src="/assets/My Logo.svg" class="logo" />
+        <img class="menu" src="assets/Side.svg" alt="menu" />
+      </nav>
+      <section class="my-section">
+        <div>
```

Comment:
- [ ] Please your code should have margins (top, bottom, left, right) defined in the design, it is a really important skill to give the same design requested from you.

| Your code  | Figma|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/211888648-01fb9d37-c7b0-40da-a8b2-97040fe910b7.png) | ![image](https://user-images.githubusercontent.com/74744787/211888440-f581d54e-5ba4-4472-bf19-1450152f720b.png)|